pfcpiface/bess_pb/module_msg.pb.go

```diff
@@ -5900,8 +5900,9 @@ type QosArg struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	Fields []*Field `protobuf:"bytes,1,rep,name=fields,proto3" json:"fields,omitempty"`
-	Values []*Field `protobuf:"bytes,2,rep,name=values,proto3" json:"values,omitempty"`
+	Fields                  []*Field `protobuf:"bytes,1,rep,name=fields,proto3" json:"fields,omitempty"`
+	Values                  []*Field `protobuf:"bytes,2,rep,name=values,proto3" json:"values,omitempty"`
+	AdjustMeterPacketLength int64    `protobuf:"varint,3,opt,name=adjust_meter_packet_length,json=adjustMeterPacketLength,proto3" json:"adjust_meter_packet_length,omitempty"`
```

Comment:
don't needs to store it here. once we add to the Protobuf, just need to run "make pb"

Reply:
I think the CI action updated it automatically.